### PR TITLE
languages: add biome support for formatting

### PR DESCRIPTION
**Pull request by nezia1**
Added [biome](https://github.com/biomejs/biome) support for formatting for Typescript, CSS and Svelte.

## Sanity Checking

[editorconfig]: https://editorconfig.org
[changelog]: https://github.com/NotAShelf/nvf/tree/main/docs/release-notes

- [x] I have updated the [changelog] as per my changes.
- [x] I have tested, and self-reviewed my code.
- Style and consistency
  - [x] I ran **Alejandra** to format my code (`nix fmt`).
  - [x] My code conforms to the [editorconfig] configuration of the project.
  - [x] My changes are consistent with the rest of the codebase.
- If new changes are particularly complex:
  - [ ] My code includes comments in particularly complex areas
  - [ ] I have added a section in the manual.
  - [ ] _(For breaking changes)_ I have included a migration guide.
- Package(s) built:
  - [x] `.#nix` (default package)
  - [x] `.#maximal`
  - [ ] `.#docs-html`
- Tested on platform(s)
  - [x] `x86_64-linux`
  - [ ] `aarch64-linux`
  - [ ] `x86_64-darwin`
  - [ ] `aarch64-darwin`

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: docs/release-notes/rl-0.7.md [hunk 1]
```diff
@@ -196,3 +196,7 @@ everyone.
 [Soliprem](https://github.com/Soliprem)
 
 - Add LSP and Treesitter support for R under `vim.languages.R`.
+
+[nezia1](https://github.com/nezia1)
```
Comment (NotAShelf) [inline docs/release-notes/rl-0.7.md:200]:
```suggestion
[nezia1](https://github.com/nezia1):
```

File: docs/release-notes/rl-0.7.md [hunk 2]
```diff
@@ -196,3 +196,7 @@ everyone.
...
+
+- Add [biome](https://github.com/biomejs/biome) support for Typescript, CSS and Svelte.
```
Comment (NotAShelf) [inline docs/release-notes/rl-0.7.md:202]: Add instructions on how to switch, you may refer to options with `[](#opt-vim.language...)`

**Review by NotAShelf — CHANGES_REQUESTED** [review]
Two quick changes, can be merged afterwards.

**Comment by NotAShelf** [comment]
Rebased on v0.7 as main is currently in feature freeze. Please fix the resolve conflict from that as well.

**Review by diniamo — APPROVED** [review]
Adding the same formatter multiple times isn't ideal, but we can't do much about it until the language rewrite anyway

**Review by NotAShelf — APPROVED** [review]

**Pull request merged**